Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.cs

```diff
@@ -1902,6 +1902,144 @@ public void When_time_is_less_than_30s_after_another_time_it_should_not_throw()
             subject.Should().BeLessThan(TimeSpan.FromSeconds(30)).After(target);
         }
 
+        [Fact]
+        public void When_asserting_subject_be_morethan_10_seconds_after_target_but_subject_is_before_target_should_throw()
```

Comment:
(Nitpicking) Missing _  between _more_ and _than_

Reply:
I was debating if i should have a `be_more_than` or `bemorethan` because the method name is `BeMoreThan` but somehow I went in the middle lol. I'll put an underscore to between the words.

Reply:
> Don’t refer to technical names such as the names of methods, variables and UI elements in the name of a test. In my opinion, the name of a test should explain the functional scenario, whereas the cause and effect should clearly illustrate the technicalities of that scenario. If some class or member is renamed, it shouldn’t affect the functional name of the test.

From https://www.continuousimprover.com/2015/11/12-tips-to-write-unit-tests-that-dont.html

In other words, I never use the literal name of a specific method in a test name.